v19.2/verify-constraint-conformance.md

```diff
@@ -0,0 +1,565 @@
+| type             | [`STRING`](string.html)       | The type of zone configuration that was violated, e.g., `constraint`.                                   |
+| config           | [`STRING`](string.html)       | The YAML key-value pair used to configure the zone, e.g., `+region=europe-west1`.                       |
+| report_id        | [`INT8`](int.html)            | The ID of the [report](#system-reports_meta) that generated this data.                                  |
+| violation_start  | [`TIMESTAMP`](timestamp.html) | The time when the violation occurred, in UTC.                                                           |
+| violating_ranges | [`INT8`](int.html)            | The [ranges](architecture/overview.html#architecture-range) that are in violation of the configuration. |
+
+### system.reports_meta
+
+{% include copy-clipboard.html %}
+~~~ sql
+SHOW COLUMNS FROM system.reports_meta;
+~~~
+
+| Column name | Data type                     | Description                                                                                      |
+|-------------+-------------------------------+--------------------------------------------------------------------------------------------------|
+| id          | [`INT8`](int.html)            | The ID of the report run that generated the data available in the `system.replication_*` tables. |
+| generated   | [`TIMESTAMP`](timestamp.html) | When the report was generated, in UTC.                                                           |
+
+### crdb_internal.zones
+
+The `crdb_internal.zones` table is useful for:
+
+- Matching up data returned from the various conformance reports with the human-readable names of the databases and tables, indexes, and partitions where that data lives.
+- Viewing your cluster's zone configurations in various formats: YAML, SQL, etc.
+
+{% include copy-clipboard.html %}
+~~~ sql
+SHOW COLUMNS FROM crdb_internal.zones;
+~~~
+
+| column_name         | data_type               | description                                                                                                          |
+|---------------------+-------------------------+----------------------------------------------------------------------------------------------------------------------|
+| zone_id             | [`INT8`](int.html)      | The ID of the [replication zone](configure-zone.html).                                                               |
+| subzone_id          | [`INT8`](int.html)      | The ID of the subzone (i.e., [partition](partition-by.html)).                                                        |
+| target              | [`STRING`](string.html) | The "object" that the constraint is being applied to, e.g., `PARTITION us_west OF INDEX movr.public.users@primary`.  |
+| range_name          | [`STRING`](string.html) | The human-readable printed representation of the range. Unimplemented.                                               |
```

Comment:
unimplemented? I think this is filled in for zones like the default one that don't refer to a database/table/...

Reply:
Ah, that was my guess based on the fact that I was only seeing `NULL` for that column.  I've removed the word "unimplemented" but did not try to add any more explanation.  Do you think anything else is needed beyond calling it "the human-readable printed representation"?